### DISCUSSION TITLE
Convergence difficulty met in LiCoO calculation with the normal nupdown setting

### DISCUSSION THREAD

**Discussion by hongriTianqi** [deepmodeling/abacus-develop#7084] [General]
### Describe the bug

Convergence difficulty met in LiCoO calculation with the nupdown setting.
The test is presented here:
https://app.bohrium.dp.tech/abacustest?request=GET%3A%2Fapplications%2Fabacustest%2Fjobs%2Fjob-abacustest-v0.3.95-eb7918

with 34/100 cases not converged.

### Expected behavior

All pass.

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

**Comment by WHUweiqingzhou** [comment]
I think this bad convergence comes from `mixing_restart=10`, which is a very bad initial guess.
I believe this issue can be solved after PR #3681 by setting `mixing_restart = 1e-3`.

**Comment by WHUweiqingzhou** [comment]
I am still working on this issue.

**Comment by rimujiangshu** [comment]
I can not open the link，so maybe you can resubmit the issue again.

**Comment by mohanchen** [comment]
Since we cannot get  the data (don't believe any link beyond github that can save the data in a considerable amount of time), I will transfer this issue to discussion
